editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -925,15 +925,16 @@ void TileMapEditor::_update_copydata() {
 
 	for (int i = rectangle.position.y; i <= rectangle.position.y + rectangle.size.y; i++) {
 		for (int j = rectangle.position.x; j <= rectangle.position.x + rectangle.size.x; j++) {
+			Vector2i ji = Vector2i(j, i);
```

Comment:
Is there a specific reason why this iterates over `y` then `x` instead of `x` then `y`?
This variable could be renamed `position`.

Reply:
I'm not sure, but this code appends to a `List<TileMapEditor::TileData>`, so changing this has the potential to break how this list is used elsewhere, so it probably belongs in another PR.

Reply:
Iterating the `y` is the outer loop, so it's actually doing it first by `x`, then incrementing the `y` after each row.  Though which order is better is entirely up to debate.  Is there a most common ordering throughout the Godot codebase?